.github/workflows/unittest.yml

```diff
@@ -0,0 +1,25 @@
+name: Unit test
+
+on: [push]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '2.8.1'
+          channel: stable
```

Comment:
```suggestion
        with:
          channel: 'stable'
```

Reply:
I had this at first, but then the tests are not passing anymore...